pages/landing/hipaa/pricing/_price-col.html

```diff
@@ -12,21 +12,22 @@ <h2>Special early access pricing</h2>
         <p class="big-price"><span>$</span>{{ include.pricing.price }} / mo<sup>*</sup></p>
         <p class="medium-price">+ $</span>{{ include.pricing.flat_fee }} flat fee</p>
       </div>
-      <div data-mh="plans-pricing-bottom">
+      <div data-mh="plans-pricing-bottom" style="margin-top: 2em;">
         <p>
-          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">I'm interested</a>
+          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Join the waitlist</a>
         </p>
         <p class="text-muted small">
-          <em>Annual contract<br>Billed monthly or annually.</em><br>* All prices are subject to change.
+          <em>* Annual contract, billed monthly or annually.<br>All prices are subject to change. No payment is needed to sign up.</em>
```

Comment:
A few thoughts:

1. "All prices are subject to change" seems to go against the, "Lock in a reduced rate for your subscription, in perpetuity!" benefit for early access subscribers. Might want to remove it.
1. Perhaps instead of "No payment is needed to sign up" we say something like "No payment is needed to join the waitlist."

Reply:
1. These two clauses are trying to say different things, but I agree they are unclear. The former is intended to make clear that they aren't locking in the prices shown on this page, which may yet change. The early access feature is a discounted rate lock in, just at a rate that may not be reflected here. How about "The price shown is not yet final"?
2. That's much clearer. Let's go with your suggestion.

Reply:
Ah, I see what you're driving at. Well, I _think_ for anyone that joins the waitlist at price X, we probably should honor price X and not change it on them... Otherwise, that really is a bait & switch. Perhaps we should even include the price as a hidden element in the form they submit so we have that data. That way, if we change it for later experiments, it'll only affect later visitors.

So do we really need the "price shown is not yet final" piece?